wiki/People/Osu!_Alumni/en.md

```diff
@@ -1,9 +1,9 @@
 osu! Alumni
-=============
+===========
 
-The **osu! Alumni** are those known for their contributions who have since moved on. Were the resources available we would erect a statue for each member in the town square. [Link](http://osu.ppy.sh/g/16)
+The **osu! Alumni** are those known for their contributions who have since moved on. Were the resources available we would erect a statue for each member in the town square. [Team Page](https://osu.ppy.sh/g/16)
```

Comment:
change `[Team Page](https://osu.ppy.sh/g/16)` to `Link to [user group page](https://osu.ppy.sh/g/16)`